### ISSUE TITLE
Content blocked on localhost

### ISSUE THREAD

**Issue by richardpenman** [gorhill/uBlock#2255]
### Describe the issue

I am developing a website and testing on localhost. I found uBlock is blocking content so disabled it for the current website, but find content is still being blocked. (Surprised anything is blocked on localhost.) Following console message is received and website does not fully load: 

    contentscript.js:79 Uncaught Error: uBlock Origin: aborting content scripts for http://localhost:8000/en/

I tried disabling uBlock for current website but content is still blocked. Then tried uninstalling uBlock and problem disappeared.

### One or more specific URLs where the issue occurs

http://localhost:8000/en/

### Screenshot in which the issue can be seen

uBlock disabled for website:

![screen shot 2016-12-21 at 09 38 02](https://cloud.githubusercontent.com/assets/1259710/21374532/549077dc-c762-11e6-9c37-b501e7b48c57.png)

### Your settings

- Browser/version: Chrome 54
- uBlock Origin version: 1.10.4

##### Your filter lists

Default filter lists

**Comment by gorhill** [comment]
Did you use the logger as asked in [CONTRIBUTING](https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md)?

**Comment by richardpenman** [comment]
Sorry, duplicate of https://github.com/gorhill/uBlock/issues/2029
